src/components/Caption/Caption.tsx

```diff
@@ -116,19 +129,23 @@ export class Caption extends Component<ExtendedCaptionProps> {
         const { text, startTime } = caption;
 
         return (
-            <div className={styles.caption}>
+            <div
+                className={styles.caption}
+                tabIndex={1}
```

Comment:
must we have 1? 
The A11Y consultant told us to try and leave things with 0 or with their natural order

Reply:
we can't use 1 until tabindex of UpperBar panel got it's own tabindex fix from core team